wpeverest/everest-forms
# Dev - Dynamic country wise state dropdown.

### All Submissions:


### Changes proposed in this Pull Request:
Previously. There was no method to select country wise state this PR enhance this feature.

### How to test the changes in this Pull Request:

1. Create a form drag and drop the
2. Drag and Drop the address Field Address field,
3. Verify the dynamic country wise state is working or not.
4. Also check the country to be selected feature is working or not in address field.

### Types of changes:




### Other information:




### Changelog entry

> Dev - Dynamic country wise state dropdown.
